Makefile

```diff
@@ -7,6 +7,15 @@ NPM=$(or $(shell which npm), $(error "Missing dependency - no npm in PATH"))
 PROTOC_IMAGE="treeverse/protoc:3.14.0"
 PROTOC=$(DOCKER) run --rm -v $(shell pwd):/mnt $(PROTOC_IMAGE)
 
+# https://openapi-generator.tech
+OPENAPI_GENERATOR_IMAGE=openapitools/openapi-generator-cli:v5.1.0
+OPENAPI_GENERATOR=$(DOCKER) run --rm -v $(shell pwd):/mnt $(OPENAPI_GENERATOR_IMAGE)
+ifndef PACKAGE_VERSION
+	PACKAGE_VERSION=0.1.0.dev
```

Comment:
Use git's SHA instead?

Reply:
Needed a constat. The build process runs `gen` and `checks-validator` which generate the client and check if there is a diff the client's API folder for changes.
This is to make sure in the PR step that you build the python client after updating the swagger API.